### DISCUSSION TITLE
High Resolution Images

### DISCUSSION THREAD

**Discussion by leenr01** [gama-platform/gama#75] [Q&A]
How to save a chart as an image with high resolution? 
Using Autosave gives me a black image.

**Comment by AlexisDrogoul** [comment]
You can try to change the value of this preference to see if it helps : 

![Image 2023-01-05 17-53-56](https://user-images.githubusercontent.com/579256/210836448-bb9b7df6-2ce1-433d-b525-f192c1634e5d.png)

**Reply by leenr01** [reply]
Unfortunately, it does not help. Even though the image size is large, the quality is still poor. Actually worse than the default.

**Comment by lesquoyb** [comment]
I think this is no longer an issue since we reworked the images a lot since then, so I'm closing. Feel free to reopen and give more details if it's still happening
